packages/theming/atlas/src/themesource/atlas_core/web/core/widgets/_badgebutton.scss

```diff
@@ -0,0 +1,38 @@
+//
+// DISCLAIMER:
+// Do not change this file because it is core styling.
+// Customizing core files will make updating Atlas much more difficult in the future.
+// To customize any core styling, copy the part you want to customize to styles/web/sass/app/ so the core styling is overwritten.
+//
+
+/* ==========================================================================
+   Badge-button
+
+   Widget styles
+========================================================================== */
+.widget-badge-button {
+  display: inline-block;
+
+  .widget-badge-button-text {
+    white-space: nowrap;
+    padding-right: 5px;
+  }
+
+  .badge {
+    display: inline-block;
+    margin: 0;
+    padding: $spacing-smaller $spacing-small;
+    text-align: center;
+    vertical-align: baseline;
+    white-space: nowrap;
+    background-color: $btn-primary-color;
+    color: $btn-primary-bg;
+    font-size: 100%;
+    line-height: 1;
+
+    .form-control-static {
+      font-weight: $font-weight-normal;
+      all: unset;
```

Comment:
What is this for?

Reply:
It resets the previous styles defined.

Reply:
Yes, MDN said the same thing 😃 What is the purpose of unsetting styles? Under what scenario is the class `form-control-static` added?

Reply:
Its a bootstrap style, then we just override specifically for this widget. It was already previously in the first version of the widget. Just moved to atlas instead.

Reply:
Good catch. Removed the whole class :)